### ISSUE TITLE
CLI exit code on error

### ISSUE THREAD

**Issue by sstarcher** [kubernetes-incubator/kompose#239]
When running `kompose -f file` for any invalid file or any invalid command I would expect a shell return code of non-zero.

**Comment by kadel** [comment]
Thank for reporting it. You are right we have to fix that

**Comment by cdrage** [comment]
@kadel @sstarcher I think this is normal behavior (with regards to not doing anything). The problem is that there's no **default** behaviour when just using ./kompose

Even I just realized that there was the "Incorrect Usage." message on the top, go figure!

```
github.com/kubernetes-incubator/kompose  master ✗                                                                            23h39m ⚑  
▶ kompose -f foobar        
NAME:
   kompose - A tool helping Docker Compose users move to Kubernetes.

USAGE:
   kompose [global options] command [command options] [arguments...]
   
VERSION:
   0.1.0 (8227684)
   
AUTHOR(S):
   Skippbox Kompose Contributors <https://github.com/skippbox/kompose> 
   
COMMANDS:
    convert     Convert Docker Compose file (e.g. docker-compose.yml) to Kubernetes objects
    up          Deploy your Dockerized application to Kubernetes (default: creating Kubernetes deployment and service)
    down        Delete instantiated services/deployments from kubernetes

GLOBAL OPTIONS:
   --file, -f "docker-compose.yml"      Specify an alternative compose file (default: docker-compose.yml) [$COMPOSE_FILE]
   --help, -h                           show help
   --generate-bash-completion
   --version, -v                        print the version
   

github.com/kubernetes-incubator/kompose  master ✗                                                                            23h39m ⚑  
▶ echo $?
0
```

Like what @sstarcher mentioned, it's suppose to be 1 :) I'll work up a patch for this.

**Comment by cdrage** [comment]
I'm trying to figure where to fix this with the most recent work in https://github.com/urfave/cli/pull/399

**Comment by cdrage** [comment]
Fixed in #304 which exits correctly with a proper error code.

```
github.com/kubernetes-incubator/kompose  switch-to-cobra ✔                                                                                                                                                                                                                   0m  
▶ ./kompose convert
ERRO[0000] Failed to find the compose file: docker-compose.yml 
FATA[0000] Failed to load compose file: open docker-compose.yml: no such file or directory 

github.com/kubernetes-incubator/kompose  switch-to-cobra ✔                                                                                                                                                                                                                  4m  ⍉
▶ echo $?
1
```
